src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -16,15 +16,15 @@
 
 namespace Coverlet.Core.Symbols
 {
-    internal static class CecilSymbolHelper
+    internal class CecilSymbolHelper : ICecilSymbolHelper
     {
         private const int StepOverLineCode = 0xFEEFEE;
-        private static ConcurrentDictionary<string, int[]> CompilerGeneratedBranchesToExclude = null;
+        private readonly ConcurrentDictionary<string, int[]> _compilerGeneratedBranchesToExclude;
 
-        static CecilSymbolHelper()
+        public CecilSymbolHelper()
         {
             // Create single instance, we cannot collide because we use full method name as key
-            CompilerGeneratedBranchesToExclude = new ConcurrentDictionary<string, int[]>();
+            _compilerGeneratedBranchesToExclude = new ConcurrentDictionary<string, int[]>();
```

Comment:
We can initialize inline and remove ctor, before we wanted to be sure single thread initialization, now we use an singleton instance so no double init issue. Keep the comment above pls.

Reply:
Done